docs/set_up_svs.rst

```diff
@@ -63,10 +63,9 @@ Key*. This operation requires the Tails administration password to be set
 
 To set the system time:
 
-#. Click the upper right down arrow in the menu bar and select the wrench icon:
-   |select settings|
-#. Select the **Details** section, then click **Date & Time**.
-#. Click **Unlock**. Type in the admin password you set when you
+#. Click in the top left menu Applications, go to **System Tools** and select Settings
```

Comment:
The buttons/targets that users will click should be bolded so they stand out as UI elements in the instructions. There are also some flow issues with the first part of the line. I might suggest a slightly tweaked version like this:

```
#. Click **Applications** in the top left of the screen, then go to **System Tools** and select **Settings**/
```